ci/tasks/shared/bump-deps.yml

```diff
@@ -22,5 +22,6 @@ params:
   GO_PACKAGE:
   GOPROXY:
   GOSUMDB:
+  GOOS_LIST: # comma-separated list of GOOS values, ex linux,darwin,windows
```

Comment:
I would recommend defaulting this to `linux`. Otherwise, every single pipeline that uses the shared task will need to be updated, rather than just updating the ones that need the multi-OS support.